BoxOfficeTests/ParseJSONDataTests.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  ParseJSONDataTests.swift
+//  ParseJSONDataTests
+//
+//  Created by LeeSeongYeon on 2024/02/13.
+//
+
+import XCTest
+@testable import BoxOffice
+
+final class ParseJSONDataTests: XCTestCase {
+    
+    func test_parseJSONData_성공시_BoxOfficeResult_인스턴스를_반환하는가() {
```

Comment:
기능 명세를 작성한다고 생각하고 테스트 이름을 지어보시면 좋겠습니다.
```suggestion
    func test_parseJSONData_성공시_BoxOfficeResult_인스턴스를_반환한다() {
```

Reply:
test 를 아예 새롭게 구성하면서, 말씀해주신 내용을 최대한 반영해보았습니다.